tests/integration/certs/tls.tf

```diff
@@ -0,0 +1,74 @@
+# this is here to conveniently generate a self-signed CA and use it to sign a cert/key pair.
+
+locals {
+  valid_hours = 24 * 365 * 50 // 50 years
+
+  files = {
+    # "ca.key"     = tls_private_key.ca.private_key_pem
```

Comment:
Should we remove this commented-out line?

Reply:
you got it!  I just amended that commit for tidy git history ⚡